docs/pages/api-docs/form-control.md

```diff
@@ -38,7 +38,7 @@ You can find one composition example below and more going to [the demos](/compon
 </FormControl>
 ```
 
-⚠️Only one input can be used within a FormControl.
+⚠️ Only one `InputBase` can be used within a FormControl because it create visual inconsistencies.
```

Comment:
Do we have a concrete example for this? Ideally we could list all so that people searching by problem find the cause.

Reply:
In https://github.com/mui-org/material-ui/issues/22535#issuecomment-689458896, we have:

![](https://user-images.githubusercontent.com/3165635/92583490-aa07b400-f292-11ea-9b31-9a7403966377.png)

Reply:
Can we describe this pattern in words?

Reply:
do i need to add this to docs?

Reply:
That'd be nice. "visual inconsistencies" is not helpful. Either we describe examples or we removeit.

Reply:
i get it from [comment](https://github.com/mui-org/material-ui/issues/22535#issuecomment-689458896), I think to delete this will be good decision if we have no other variants